src/content/json/teams.ts

```diff
@@ -0,0 +1,91 @@
+import TeamsTitleImage from '../images/teams_title_block.png';
+
+const teams_content = {
```

Comment:
We should use camel case for the variable and property names:
`const teamsContent = {  
  titleBlock: { ...
`

Reply:
oof good catch ty!!